rbs/ip.tf

```diff
@@ -0,0 +1,7 @@
+resource "google_compute_address" "ingress_ip" {
+  count = var.enable ? 1 : 0
+
+  provider = google.target
+
+  name = "terra-${var.cluster}-${local.owner}-${local.service}-ip"
```

Comment:
Can we file/TODO a ticket to follow up on the ingress/firewall we want for RBS? Eventually we need ingress for Rawls (as long as it's not in the same k8s cluster).

Reply:
We have a Jira ticket RBS service authZ. But I don't think that TODO belongs to here. Here is to generate an ingress_ip address. 
Also, testing tools RBS will need this even Rawls in the same cluster. 
What would be the TODO here?

Reply:
sg, no todo.